x/concentrated-liquidity/lp.go

```diff
@@ -1,6 +1,32 @@
 package concentrated_liquidity
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	types "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types"
+)
+
+func (k Keeper) JoinPool(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokenIn sdk.Int, lowerTick sdk.Int, upperTick sdk.Int) (numShares sdk.Int, err error) {
```

Comment:
Wdyt about changing `JoinPool` to `Mint` or `ProvideLiquidity` since we've discussed wanting to incrementally move away from the pool abstraction in concentrated-liquidity

Reply:
will go with Mint for now, we can decide on which method name would suit the best later on